cost-analyzer/values.yaml

```diff
@@ -173,6 +173,11 @@ saml:
         assertionName:  "http://schemas.auth0.com/userType"
         assertionvalues:
           - "readonly"
+      - name: editor
+        enabled: true # if editor is enabled, editors will be allowed to edit reports/alerts scoped to them, and act as readers otherwise. Users will never default to editor.
```

Comment:
Might start with this false? otherwise LGTM

Reply:
Ah yep, should be false.